libs/vkd3d/command.c

```diff
@@ -5489,20 +5473,20 @@ static void d3d12_command_list_update_root_descriptors(struct d3d12_command_list
 
             vk_write_descriptor_set_from_root_descriptor(list,
                     &descriptor_writes[descriptor_write_count], root_parameter,
-                    descriptor_set, &bindings->root_descriptors[root_parameter_index]);
+                    VK_NULL_HANDLE, &bindings->root_descriptors[root_parameter_index]);
 
             descriptor_write_count += 1;
         }
 
         bindings->root_descriptor_dirty_mask = 0;
     }
 
-    if (root_signature->flags & VKD3D_ROOT_SIGNATURE_USE_INLINE_UNIFORM_BLOCK)
+    if (root_signature->flags & VKD3D_ROOT_SIGNATURE_USE_PUSH_CONSTANT_UNIFORM_BLOCK)
     {
-        d3d12_command_list_fetch_inline_uniform_block_data(list, bindings, &root_parameter_data);
-
-        vk_write_descriptor_set_and_inline_uniform_block(&descriptor_writes[descriptor_write_count],
-                &inline_uniform_block_write, descriptor_set, root_signature, &root_parameter_data);
+        d3d12_command_list_fetch_inline_uniform_block_data(list, bindings, ptr_root_parameter_data);
```

Comment:
Might actually want to rename `d3d12_command_list_fetch_inline_uniform_block_data` to... something as well since we're no longer using inline uniform blocks.